src/reconcile/rec_write.c

```diff
@@ -1988,13 +1988,10 @@ __rec_split(WT_SESSION_IMPL *session, WT_RECONCILE *r, size_t next_len)
 		 * add whatever space remains in this page.
 		 */
 		if (inuse + r->split_size <= r->page_size)
-			r->space_avail =
-			    r->split_size - WT_PAGE_HEADER_BYTE_SIZE(btree);
+			r->space_avail = r->split_size;
```

Comment:
Hold off on the merge: I'm rethinking this half of the change.  We probably do want to allow for a header in case this new chunk is eventually written.